virtool/account/api.py

```diff
@@ -352,6 +352,10 @@ async def get(self) -> r204:
             virtool.http.authentication.get_ip(self.request),
         )
 
+        await self.request.app["ws"].close_connections_with_session_id(
+            self.request.cookies.get("session_id")
+        )
```

Comment:
While this works, it tightly couples the WS server to the API, ideally we would disconnect the user via a message over redis